skin/feedlist.css

```diff
@@ -39,8 +39,9 @@ tree-folder-header > .toggle-collapsed::-moz-focus-inner {
 
 tree-folder:not(.unread) > tree-folder-header > .unread-count,
 tree-folder:not(.collapsed) > tree-folder-header > .unread-count,
-tree-item:not(.unread) > .unread-count {
-    visibility: hidden;
```

Comment:
The unread count was intentionally `visibility: hidden;` instead of `display: none;` to make the text cutoff points better aligned. Compare the following screenshots:
![Screenshot_20190603_005838](https://user-images.githubusercontent.com/1027034/58767904-bb93e380-859a-11e9-8917-595323ee2c81.png) - `display: none;`
![Screenshot_20190603_005947](https://user-images.githubusercontent.com/1027034/58767913-eb42eb80-859a-11e9-8840-d3e7e3c956fa.png) - `visibility: hidden;`